languages/elixir/exercises/concept/rpg-character-sheet/.docs/instructions.md

```diff
@@ -0,0 +1,61 @@
+You and your friends love to play pen-and-paper role-playing games, but you noticed that it's difficult to get new people to join your team. They often struggle with character creation. They don't know where to start. To help new players out, you decided to write a small program that will guide them through the process.
+
+## 1. Welcome the new player
+
+Implement the `RPG.CharacterSheet.welcome/0` function. It should print a welcome message, and return `:ok`.
+
+```elixir
+RPG.CharacterSheet.welcome()
+# Welcome! Let's fill out your character sheet together.
+# => :ok
+```
+
+## 2. Ask for the character's name
+
+Implement the `RPG.CharacterSheet.ask_name/0` function. It should print a question, wait for an answer, and return the answer without leading and trailing whitespace.
+
+```elixir
+RPG.CharacterSheet.ask_name()
+# What is your character's name?
+# Mathilde
+# => "Mathilde"
+```
```

Comment:
I don't know how to mark input and output in those code examples...

Reply:
maybe just a single 'greater than sign. (`>`)?

```
# What is your character's name?
# > Mathilde
```

Reply:
I tried adding `>` for output and `<` for input, what do you think?

Reply:
I think it communicates the intention.  I am unaware of any real standard for this, so I think it works. @jwarwick thoughts?